packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -260,8 +260,9 @@ function UserMenu({
     },
   };
 
-  const urls: URLSInterface = defaultsDeep(urlsProp, defaultURLs);
+  const urls: Required<URLSDefaults> = defaultsDeep(urlsProp, defaultURLs);
   const userMenu = urls.userMenu ?? {};
+  const cloudUrls: Partial<UserMenuURLS['userMenu']['cloud']> = userMenu.cloud;
```

Comment:
Lets us remove the existence checks below.

```suggestion
  const cloudUrls: Partial<UserMenuURLS['userMenu']['cloud']> = userMenu.cloud ?? {};
```

Reply:
Took me an embarrassingly long time to find a typing that would support it but arrived at a very simple conclusion 😥